public/js/storeFront.js

```diff
@@ -3,133 +3,70 @@ document.addEventListener("DOMContentLoaded", event => {
     console.info("DOM loaded");
   }
 
-  document.getElementById(".cartBtn").addEventListener("submit", e => {
-    e.preventDefault();
-    fetch("api/storefront", {
-      method: "PUT",
-      headers: {
-        Accept: "application/json",
-        "Content-Type": "application/json"
-      }
-    }).then(data => {
-      console.log(data);
-    });
-  });
-
-  //   $(".cartBtn").on("click", () => {
-  //     //call event in brackets.
-  //     //const id = $(this).data("id");
-  //     fetch(`/api/storefront/${id}`, {
-  //       method: "PUT",
-  //       headers: {
-  //         Accept: "application/json",
-  //         "Content-Type": "application/json",
-  //       }.then(() => {
-  //         console.log("added to cart"); /// Success response
-  //         location.reload();
-  //       }),
-  //     });
-  //   });
-
-  //   $(".cartBtn").on("click", function() {
-  //     //call event in brackets.
-  //     const id = $(this).data("id");
-  //     $.ajax("/api/cart" + id);
-  //     $.ajax({
-  //       type: "POST",
-  //       url: url, //appropriate url.... apicart/id
-  //       data: productId
-  //     }).then(() => {
-  //       console.log("added to cart"); /// Success response
-  //       location.reload();
-  //     });
-  //   });
-
-  //   const addToCartButton = document.querySelectorAll(".devoured");
+  const cartBtn = document.querySelectorAll(".cartBtnH");
```

Comment:
Is the button class actually ".cartBtnH" with an H or was this a typo?

Reply:
not  a typo, just to remind me its handlebars when writing js